test/rubber_tree.md

```diff
@@ -0,0 +1,9 @@
+# Rubber Tree
+
+1. Run client.
+2. Create a new world (creative world).
+3. Fly around to find a rubber tree.
+4. Dig all the logs and check whether the leaves break automatically and drop rubber tree sapling.
+   (You might be very unlucky that rubber tree sapling don't drop but in fact the mod works well, so you should replace this several times before report a bug).
+5. Plant the sapling and wait it growing.
```

Comment:
Actually bone powder works！

Reply:
If natual growing works, bone powder also works, but not vice versa.
MC is really strange……